recipes/micro/meta.yaml

```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "micro" %}
+{% set version = "1.3.3" %}
+{% set sha256 = "329f746e4ee9edf244618dda4208b638fda34c593d5cd96d8f71dc3b53e3d994" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+#  fn: {{ name }}-{{ version }}.tar.g
+  url: https://github.com/zyedidia/micro/releases/download/v1.3.3/micro-{{ version }}-linux64.tar.gz
```

Comment:
It looks like you need to use selectors and specify a different tarball for each platform:

https://github.com/zyedidia/micro/releases

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors

Or perhaps you should be building from source.

Reply:
Done